specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -692,78 +692,78 @@ model AcsRouterWorkerSelector {
   key?: string;
 
   /** Router Job Worker Selector Label Operator */
-  operator?: AcsRouterLabelOperator;
+  operator: AcsRouterLabelOperator;
 
   /** Router Job Worker Selector Value */
   #suppress "@azure-tools/typespec-azure-core/no-unknown" "The type of labelValue is object"
-  labelValue?: unknown;
+  labelValue: unknown;
 
   /** Router Job Worker Selector Time to Live in Seconds */
   @encode("ISO8601") // TODO verify this is timespan in C#
-  ttlSeconds?: duration;
+  ttlSeconds: duration;
 
   /** Router Job Worker Selector State */
-  selectorState?: AcsRouterWorkerSelectorState;
+  selectorState: AcsRouterWorkerSelectorState;
```

Comment:
Looks like this is actually under 'state', not 'selectorState':

https://github.com/Azure/azure-rest-api-specs/blob/44cbf51b9eee8a2dbf55ced61a61032aaed0148b/specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json#L2171